### PR TITLE
Initial implementation of gamelab cartridges

### PR DESCRIPTION
**Pull request by maxwofford**

### CODE REVIEW

**Comment by maxwofford** [comment]
This is linked to https://github.com/hackclub/scrappy/pull/306

File: public/cartridge.css [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+@font-face {
+  font-family: "Phantom Sans";
+  src: url("https://assets.hackclub.com/fonts/Phantom_Sans_0.7/Bold.woff")
+      format("woff"),
+    url("https://assets.hackclub.com/fonts/Phantom_Sans_0.7/Bold.woff2")
+      format("woff2");
+  font-weight: bold;
+  font-style: normal;
+  font-display: swap;
+}
```
Comment (sampoder) [inline public/cartridge.css:23]: Can we remove these?

File: public/cartridge.css [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+  src: url("https://assets.hackclub.com/fonts/Phantom_Sans_0.7/Regular.woff")
+      format("woff"),
+    url("https://assets.hackclub.com/fonts/Phantom_Sans_0.7/Regular.woff2")
+      format("woff2");
+  font-weight: normal;
+  font-style: normal;
+  font-display: swap;
+}
+
+:root {
```
Comment (sampoder) [inline public/cartridge.css:69]: Could we use theme-ui's colour variables?

File: public/cartridge.css (hunk 1 above)
Comment (maxwofford) [reply public/cartridge.css:23]: Actually, i just checked this again-- we don't have phantom sans on scrapbook

File: public/cartridge.css [hunk 3]
```diff
@@ -0,0 +1,215 @@
+/* grabbed from https://replit.com/@MaxWofford/cartridge-experiment#cartridge/style.css */
```
Comment (sampoder) [inline public/cartridge.css:1]: Could this be rewritten to use CSS Modules (add be put in `components`) or inline styles? I'd like to help sort the repo's files

**Pull request merged**